### DISCUSSION TITLE
Problem in exiting the site appveyor

### DISCUSSION THREAD

**Discussion by mehrdadpirnazar** [artisan-roaster-scope/artisan#1579] [Q&A]
Hello good time
I am getting this error related to the one I sent the picture of
I have also set Github token in the settings.
![Screenshot 2024-06-10 110313](https://github.com/artisan-roaster-scope/artisan/assets/82454404/b2f7b73a-70c6-416a-92a4-423fbb9ba607)

**Comment by roasterdave** [comment]
Duplicate of #1577
